src/hardware/StandardLED.cpp

```diff
@@ -7,9 +7,7 @@
 #include "GPIOPin.h"
 #include "StandardLED.h"
 
-StandardLED::StandardLED(GPIOPin& gpioInstance) : gpio(gpioInstance) {
-
-}
+StandardLED::StandardLED(GPIOPin& gpioInstance) : gpio(gpioInstance) {}
```

Comment:
The curly went one line to far up. 😆

Reply:
You sure? :D

I'd suggest let's leave the individual styling comments for after this MR, see my comment above: I am suggesting a proper `clang-format` setup with `pre-commit` and GH actions - then we don't have to bother with nuisances like bracket positions anymore

Reply:
As long as we don't want to chance our code style I'm sure :p